src/main/app/src/app/shared/material-form-builder/form-components/autocomplete/autocomplete-validators.ts

```diff
@@ -47,7 +47,8 @@ export class AutocompleteValidators {
             } else if (object1.hasOwnProperty('name')) {
                 return object1.name === object2.name;
             }
-            return object1 === object2;
+
+            return JSON.stringify(object1) === JSON.stringify(object2);
```

Comment:
we hebben expliciet hier voor een === vergelijking gekozen om dat JSON.stringify  nogal de performance omlaag trekt met grote objecten.  Maar om het fail fast principe te hanteren is het misschien handiger om hier een error te gooien dat er niet een field beschikbaar is om mee te vergelijken.

Aangezien het hier om zaaktype gaat kunnen we kijken welke attributen hij heeft en blijkbaar heeft hij geen van de bovenstaande. Dus mijn voorstel is om het else if lijstje uit te breiden met de uuid en  het identifier field. Zo ondervangen we wellicht ook toekomstige issues.